### ISSUE TITLE
Sample app start problem

### ISSUE THREAD

**Issue by ktrzeciaknubisa** [jxcore/jxcore-cordova#3]
Once in a while there is a problem with application startup. I'm using iOS Simulator (iPhone 6, but I've tested also few lower). I'm talking about unmodified sample taken from this repo, however for the first time I noticed this behavior while working on my own piece of the code.

Normally, when it's all good, the log displays something like this:

``` txt
Initializing JXcore engine
JXcore engine is ready
Starting JXcore engine
JXcore Cordova Bridge is Ready!
JXcore engine is started
2015-03-10 18:31:27.168 HelloWorld[9530:523206] Application On Resume

2015-03-10 18:31:27.171 HelloWorld[9530:523206] Is Ready 1
loadMainFile app.js
getBuffer is called!!!
```

However from time to time it ends up on "Application On Resume" line, so the last 3 lines are never printed. On client's side `jxcore.isReady()` is never called.

To reproduce this I just press "stop" and "play" buttons on Xcode navbar without changing anything in the project. It occurs randomly, maybe only 2 per 10 times.

**Comment by obastemur** [comment]
this must be a timing issue. i.e. client side scripts act before jxcore.js is available etc. You can try adding some extra trace on both client and server side to catch this.

**Comment by deefactorial** [comment]
I have found that with cordova apps if there is a lot of script files loading on the index page sometimes the ondevice ready function doesn't get called. The solution is to redirect to another page immediately. Then load everything on the other page to prevent the race condition.

Index.html

``` html
<!doctype html>
<html>
  <head>
   <title>Example</title>
   <script>
     window.location='./main.html';
   </script>
  <body>
  </body>
</html>
```

Here is a reference I found:
https://www.robertkehoe.com/2013/01/fix-for-phonegap-connection-to-server-was-unsuccessful/
